### DISCUSSION TITLE
GTR 3, impossible to update firmware

### DISCUSSION THREAD

**Discussion by TyaglovOV** [zepp-health/zeppos-forum#322] [Developer Feedback]
Hello.

I started implementing some app in simulator, and faced a problem -- in simulator I can select firmware 1.0.5 for GTR 3, and it contains methods `hmSetting.stopWristDownOff` and `hmSetting.releaseWristDownOff`, which is crucial for my app, but they are available only for v1.0.5. Unfortunately, my watch isn't v1.0.5 (software version 7.47.5.1, algorithm version 16.8.5, Zepp OS `1.0`), so it's impossible for me implement this functionality.
I've tried update firmware through zepp app on phone and it said that I have latest version of OS installed, I've tried hard reset on the watch and reinstalled app on the phone -- it was pointless.

Maybe you can provide me with some information about updating firmware in this case?

Thank you.

**Comment by TyaglovOV** [comment]
On the other hand, if you provide me with information how to prevent watches from going to sleep on v.1.0, I'll be grateful.
I've tried `hmApp.setScreenKeep`, `hmSetting.setBrightScreen` -- nothing works. But it's definitely possible, cause default flashlight app  prevents watches from going to sleep, when wrist is down.

Thank you.

**Comment by MyoungXUE** [comment]
Sorry, the Zepp OS version of Amazfit GTR3  is 1.0, this version does not provide the relevant interface to close the wrist screen, so you can not achieve the effect you want. 

We have opened the pauseDropWristScreenOff interface in Zepp OS 2.1, and devices with Zepp OS 2.1 and above can achieve the effect you want.

**Reply by TyaglovOV** [reply]
Oh, okay.
Thank you for reply.
